emcc

```diff
@@ -109,7 +109,8 @@ stderr = PIPE if not DEBUG else None # unless we are in DEBUG mode
 EMCC_CXX = '--emscripten-cxx' in sys.argv
 sys.argv = filter(lambda x: x != '--emscripten-cxx', sys.argv)
 
-shared.check_sanity(force=DEBUG)
+if len(sys.argv) != 1 and '--help' not in sys.argv and sys.argv[1] != '--version':
+  shared.check_sanity(force=DEBUG)
```

Comment:
this means we don't get sanity checks on `emcc --version`, `emcc --help`? We sometimes suggest people run those commands in fact to do a sanity check, so this worries me.

i think it's better to not do this, and let it fail if the person is forcing fastcomp off.